i18n-helpers/src/bin/mdbook-gettext.rs

```diff
@@ -107,10 +55,14 @@ fn preprocess() -> anyhow::Result<()> {
 fn main() -> anyhow::Result<()> {
     if std::env::args().len() == 3 {
         assert_eq!(std::env::args().nth(1).as_deref(), Some("supports"));
-        if let Some("xgettext") = std::env::args().nth(2).as_deref() {
-            process::exit(1)
+        if let Some(renderer) = std::env::args().nth(2).as_deref() {
+            let gettext = Gettext;
+            if gettext.supports_renderer(renderer) {
```

Comment:
I can't change the definition of `supports_renderer` and `run` because they are defined as methods of `Preprocessor` trait at https://docs.rs/mdbook/latest/mdbook/preprocess/trait.Preprocessor.html.

I can define another static method like `impl_supports_renderer`, and change it to `Gettext::impl_supports_renderer`,
but it may be redundant.

Reply:
Ah, I see now! The syntax I used above would only work if the `supports_renderer` method had no `&self` argument.

Then let's merge this PR as it is — and then later change the code here to create a single `Gettext` value and pass this around. That would be a little more natural, I think (thought it would just be a small refactor).